src/styles/index.scss

```diff
@@ -1,3 +1,235 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap';
+
+$primary-color: #00acdc;
+$dark: #060b35;
+$blue: #00acdc;
```

Comment:
The `$blue` variable is assigned the same color as `$primary-color`. Consider removing one of these variables to avoid redundancy.